content/blog/accessing-dtap-in-pods.md

```diff
@@ -0,0 +1,211 @@
+---
+title: Getting Started with DataTaps in Kubernetes Pods
+date: 2021-07-06T06:44:24.016Z
+author: Cenz Wong
+authorimage: https://avatars.githubusercontent.com/u/44856918?v=4
+---
+## What is DataTap?
```

Comment:
Cenz, I noticed that you seemed to have a number of issues in the formatting of this post. I've tried to do my best to review it, but there were some areas where it was difficult to know whether something was a sentence, an instruction, a piece of code, or a header. If you want to make the changes that Denis and I suggest here and then have us take one more look at it, we'd be happy to do so.

Reply:
Hi, Cenz. Thank you for working on the formatting. Please mark this issue as resolved, as a part of the review process. Thank you!